cpp/include/torch_tensorrt/macros.h

```diff
@@ -11,8 +11,9 @@
 #define TORCHTRT_API __attribute__((__visibility__("default")))
 #define TORCHTRT_HIDDEN __attribute__((__visibility__("hidden")))
 #else
-#define TORCHTRT_API
-#define TORCHTRT_HIDDEN
+// This can be used also if defined(__GNUC__) but is generated by CMake
+// and will be missing when compiling with Bazel
+#include <torch_tensorrt_export.h>
```

Comment:
Can we #ifdef this based on CMake being the build system if it is not going to be generated by bazel?

Reply:
Yes absolutely. Very good point. This is now done in [this commit](https://github.com/pytorch/TensorRT/pull/1058/commits/bc693d86f5c52958a5cb4b34262e0761a7ed2d4a).